homeassistant/components/homekit/type_locks.py

```diff
@@ -76,6 +96,58 @@ def __init__(self, *args: Any) -> None:
             setter_callback=self.set_state,
         )
         self.async_update_state(state)
+        self.init_doorbell(self)
+
+    def init_doorbell(self) -> None:
```

Comment:
We should make this a mixin or base class so we don't have the same code in cameras

Reply:
put the code into util.py?

Reply:
Maybe `doorbell_mixin.py` or `doorbell_base.py`

Reply:
I did the changes, but I am not that familiar with python and I have also currently no possibility to test it locally. Or, to be honest, I don't know how 😅

Reply:
https://developers.home-assistant.io/docs/development_environment

Please read the docs on setting up a dev environment.

Reply:
You can also test on a live system with
```
cd /config
curl -o- -sSL https://gist.githubusercontent.com/bdraco/43f8043cb04b9838383fd71353e99b18/raw/core_integration_pr | bash /dev/stdin -d homekit -p 131660
```

Reply:
@bdraco It's working for me with my local HA running on an raspberry pi 5